excludes.txt

```diff
@@ -0,0 +1,19 @@
+# TODO cryptic PCT-only errors: https://github.com/jenkinsci/bom/pull/251#issuecomment-645012427
+hudson.matrix.AxisTest
+
+# TODO fails for one reason in (non-PCT) official sources, run locally; and for another reason in PCT in Docker; passes in official sources in Docker, or locally in PCT
+org.jenkinsci.plugins.gitclient.FilePermissionsTest
+
+# TODO flakes on CI for inscrutable reasons
+org.jenkinsci.plugins.durabletask.BourneShellScriptTest
+org.jenkinsci.plugins.pipeline.modeldefinition.BasicModelDefTest
+org.jenkinsci.plugins.pipeline.modeldefinition.MatrixTest
+
+# TODO https://github.com/jenkinsci/workflow-multibranch-plugin/pull/128
+org.jenkinsci.plugins.workflow.multibranch.JobPropertyStepTest
+
+# TODO until dropping 2.235.x so can rely on https://github.com/jenkinsci/workflow-basic-steps-plugin/pull/120
+org.jenkinsci.plugins.workflow.steps.TimeoutStepTest
```

Comment:
will be removed after release done, see https://github.com/jenkinsci/bom/pull/609

(don't remove yet)
```suggestion
```